packages/styles/docs/sass.md

```diff
@@ -89,10 +99,23 @@ between version updates.
 
 ## Motion
 
+The motion package provides helper functions, mixins, and duration tokens to add motion into your project.
+
+You can bring in the motion package and tokens by using `@use`. 
+
+For example:
+
```

Comment:
I think we can drop these kinds of snippets and just reference the snippet directly 👀 

```suggestion
```